applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -355,7 +355,7 @@ config BUTTON_MSG_SUB_STACK_SIZE
 
 config LE_AUDIO_MSG_SUB_STACK_SIZE
 	int "Stack size for LE Audio subscriber"
-	default 1250
+	default 2048
```

Comment:
This seems like a fairly large increase.
Is it necessary to increase it that much?

Reply:
The stacj usage was 1350, so about 65% usage. That leaves us some room for additional variables etc in the future